openbb_terminal/miscellaneous/data_sources_default.json

```diff
@@ -6,7 +6,7 @@
     "candle": [],
     "codes": ["Polygon"],
     "news": ["Feedparser", "NewsApi"],
-    "load": ["YahooFinance", "IEXCloud", "AlphaVantage", "Polygon", "EODHD"],
+    "load": ["YahooFinance", "IEXCloud", "AlphaVantage", "Polygon", "EODHD", "Intrinio"],
```

Comment:
Not sure why, but I needed to add the source manually on `OpenBBUserData/data_sources_default.json` in order to use it